test/fixtures/output/iam/rbac/role-binding/list-user-orgadmin-cloud.golden

```diff
@@ -1,5 +1,6 @@
-     Principal    |      Name       |        Email          
-------------------+-----------------+-----------------------
-  User:pool-12345 | identity-pool   |                       
-  User:sa-12345   | service-account |                       
-  User:u-11aaa    |                 | u-11aaa@confluent.io  
+     Principal    |         Name          |        Email          
+------------------+-----------------------+-----------------------
+  User:group-abc  | another-group-mapping |                       
+  User:pool-12345 | identity-pool         |                       
+  User:sa-12345   | service-account       |                       
+  User:u-11aaa    |                       | u-11aaa@confluent.io  
```

Comment:
What about combining the "Name" and "Email" column in v4?

Reply:
That's an option. The other option is that we can just display the user's name here; I'm not sure why we currently aren't.

Reply:
Oh, I don't think that field was added until a year or so ago. We should definitely add that, but removing the email might cause problems.